channels/ar.m3u

```diff
@@ -148,3 +148,7 @@ http://live-edge01.telecentro.net.ar/live/smil:trm.smil/playlist.m3u8
 http://streamspub.manasat.com:1935/tvar/tvmanaar2/playlist.m3u8
 #EXTINF:-1 tvg-id="TVPublica.ar" tvg-country="AR" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/0/0a/Logo_Televisi%C3%B3n_P%C3%BAblica_Argentina.png" group-title="General",TV Pública (TVP) (1080p) [Not 24/7]
 http://free.fullspeed.tv/query?url=https://www.youtube.com/user/TVPublicaArgentina/live
+#EXTINF:-1 tvg-id="Vorterix.ar" tvg-country="AR" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/vorterix/picture?width=320&height=320" group-title="Entertainment",Vorterix [Not 24/7]
+http://free.fullspeed.tv/query?url=https://www.youtube.com/channel/UCvCTWHCbBC0b9UIeLeNs8ug/live
+#EXTINF:-1 tvg-id="Vorterix.ar" tvg-country="AR" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/vorterix/picture?width=320&height=320" group-title="Entertainment",Vorterix [Not 24/7]
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCvCTWHCbBC0b9UIeLeNs8ug/live
```

Comment:
```suggestion
https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.youtube.com/channel/UCvCTWHCbBC0b9UIeLeNs8ug/live
```

Will need to check my Heroku, for now you can use my REPL link.

Reply:
Thank you.